manager/integration/tests/test_scheduling.py

```diff
@@ -611,18 +611,12 @@ def test_data_locality_basic(client, core_api, volume_name, pod, settings_reset)
     pod1['spec']['nodeSelector'] = \
         {"kubernetes.io/hostname": volume1_attached_node}
     create_and_wait_pod(core_api, pod1)
-
-    wait_for_rebuild_start(client, volume1_name)
-
-    volume1 = client.by_id_volume(volume1_name)
-    assert len(volume1.replicas) == 2
-
-    wait_for_rebuild_complete(client, volume1_name)
-
     volume1 = wait_for_volume_replica_count(client, volume1_name, 1)
     volume1 = client.by_id_volume(volume1_name)
     assert len(volume1.replicas) == 1
     assert volume1.replicas[0]['hostId'] == volume1_attached_node
+    delete_and_wait_pod(core_api, pod1_name)
+    wait_for_volume_detached(client, volume1_name)
```

Comment:
Make sure to clean up the pod after we done with it. Otherwise, PVC cannot be removed at the in the finalized at the end of the test.